pkg/chart/fake/chart.go

```diff
@@ -32,13 +32,15 @@ func (f *FakeChart) ParseDetails(data []byte) (*chartUtils.Details, error) {
 	return details, err
 }
 
-func (f *FakeChart) GetChart(details *chartUtils.Details, netClient chartUtils.HTTPClient) (*chart.Chart, error) {
-	return &chart.Chart{
-		Metadata: &chart.Metadata{
-			Name: details.ChartName,
-		},
-		Values: &chart.Config{
-			Raw: details.Values,
+func (f *FakeChart) GetChart(details *chartUtils.Details, netClient chartUtils.HTTPClient) (*chartUtils.ChartMultiVersion, error) {
+	return &chartUtils.ChartMultiVersion{
```

Comment:
Should there be no `V3` in this `ChartMultiVersion`?

Reply:
Currently the `FakeChart` is only used in a test which is testing the HelmV2 charts, so I didn't add dead code. We can add a V3 if/when the `FakeChart[Client]` is used with HelmV3 charts.